docs/autostart_win.rst

```diff
@@ -0,0 +1,105 @@
+.. windows service guide
+
+==================================
+Setting up auto-restart on Windows
+==================================
+
+-------------------------------
+Downloading NSSM via Chocolatey
+-------------------------------
+
+.. note:: This assumes you already have chocolatey installed. 
+          If not, you may install it or follow the manual instructions for getting NSSM.
+
+To install via Chocolatey, search "powershell" in the start menu,
+right-click on it and then click "Run as administrator"
+
+Then run the following command:
+
+.. code-block:: none
+
+    choco install nssm -y
+
+------------------------
+Manually installing NSSM
+------------------------
+
+Go to `<https://nssm.cc/download>`_ and download the latest release. Once the download
```

Comment:
It might be good to point at latest **pre** release (or at least pre-release build 2.2.4-101) as otherwise users may have problems with starting service on Windows 10 1703+. Technically we avoid this issue with `AppNoConsole=1` but choco already installs that build anyway and this build also comes with handy `nssm dump` command that may help certain things easier for Support Squad.